torchvision/transforms/transforms.py

```diff
@@ -1699,3 +1699,43 @@ def _setup_angle(x, name, req_sizes=(2, )):
         _check_sequence_input(x, name, req_sizes)
 
     return [float(d) for d in x]
+
+
+class RandomInvert(torch.nn.Module):
+    """Inverts the colors of the given image randomly with a given probability.
+    The image can be a PIL Image or a torch Tensor, in which case it is expected
+    to have [..., H, W] shape, where ... means an arbitrary number of leading
+    dimensions
+
+    Args:
+        p (float): probability of the image being color inverted. Default value is 0.5
+    """
+
+    def __init__(self, p=0.5):
+        super().__init__()
+        self.p = p
+
+    @staticmethod
+    def get_params() -> float:
+        """Choose value for random color inversion.
+
+        Returns:
+            float: Random value which is used to determine whether the random color inversion
+            should occur.
+        """
+        return torch.rand(1).item()
+
+    def forward(self, img):
+        """
+        Args:
+            img (PIL Image or Tensor): Image to be inverted.
+
+        Returns:
+            PIL Image or Tensor: Randomly color inverted image.
+        """
+        if self.get_params() < self.p:
```

Comment:
We have decided on getting random apply param with static `get_params()` ?

Reply:
I've asked @fmassa and he recommended adding it in the `get_params()`.

Reply:
The output of `get_params` should in principle be the parameters we will be using to apply a transformation.
In this case, it would be `True / False` for applying the transform or not.

Reply:
@fmassa OK and what to do if there are both to sample: transform random params and random apply one ?

Reply:
I don't have a good solution for this one yet. It's a similar story as `ColorAdjust`, and the solutions are not great.

We might need to revisit our transforms story, maybe breaking it down into two types of transforms.